### PR TITLE
Adjust get plugin settings

### PR DESCRIPTION
**Pull request by Alexander-Dubrawski**
Since the `meta_settings` table inside the Hyrise was adjusted we needed to adjust the cockpit as well.

## Todo

- [x] Update hyrise docker container

### CODE REVIEW

File: tests/database_manager/test_database.py [hunk 1]
```diff
@@ -718,11 +723,11 @@ def test_gets_plugins_settings_when_database_unblocked_and_plugins_exist(
 
         expected = {
             "Compression": [
-                {"name": "MemoryBudget", "value": "55555", "description": "..."}
+                {"name": "MemoryBudget (MB)", "value": "55555", "description": "..."}
```
Comment (Bouncner) [inline tests/database_manager/test_database.py]: Space missing.

File: tests/database_manager/test_database.py [hunk 2]
```diff
@@ -706,8 +706,13 @@ def test_gets_plugins_settings_when_database_unblocked_and_plugins_exist(
         """Test get existing plug-ins settings."""
         mock_cursor = MagicMock()
         mock_cursor.fetchall.return_value = [
-            ("Plugin::Compression::MemoryBudget", "55555", "..."),
-            ("Plugin::Something::SomeSetting", "true", "this should show up"),
+            ("Plugin::Compression::MemoryBudget", "55555", "...", "MemoryBudget (MB)"),
```
Comment (Bouncner) [inline tests/database_manager/test_database.py]: Space ...

File: tests/database_manager/test_database.py (hunk 1 above)
Comment (Bensk1) [reply tests/database_manager/test_database.py]: Right, `Memory Budget (MB)`.

File: hyrisecockpit/database_manager/database.py [hunk 3]
```diff
@@ -284,11 +284,18 @@ def _get_plugins(self) -> Optional[List[str]]:
             return [row[0].split("Plugin")[0] for row in rows]
 
     def _get_plugin_setting(self) -> Plugins:
-        """Return currently set plugin settings."""
+        """Return currently set plug-in settings.
+
+        The plug-in settings are obtained from the meta_settings tables in the hyrise instance.
```
Comment (Bensk1) [inline hyrisecockpit/database_manager/database.py]: table.

File: hyrisecockpit/database_manager/database.py [hunk 4]
```diff
@@ -284,11 +284,18 @@ def _get_plugins(self) -> Optional[List[str]]:
...
+        Inside the hyrise instance not just plug-ins have settings. As a result we need to differentiate
+        the plug-in settings from the other settings. Thats why we use the SQL Statement WHERE name LIKE 'Plugin::%';.
+        To get the plug-in name with which we communicate in the cockpit we need to extract it from the name column with
+        row[0].split("::")[1]. So for example from Plugin::Compression::MemomrySetting we get Compression.
```
Comment (Bensk1) [inline hyrisecockpit/database_manager/database.py]: Memomry

**Review by Bensk1 — CHANGES_REQUESTED** [review]

**Comment by Bensk1** [comment]
I thought the docker container is automatically built when the Hyrise branch `bp1920` is updated.

**Comment by Alexander-Dubrawski** [comment]
> I thought the docker container is automatically built when the Hyrise branch `bp1920` is updated.

At the moment we are building it manually. We could of course adjust Jenkins to build it for us, but actually I think it is easier to build it manually.  For example, if we change something in the Hyrise (rename the columns in a table) it could result in a failure of the cockpit. So it is easier to make the adjustments in the cockpit and than building the image manually.

**Review by Bouncner — APPROVED** [review]
LGTM.

**Comment by Bensk1** [comment]
Manually building it is fine for me for now. But I somehow thought that @cH3n7i said a new docker image would be built with each push to `bp1920`.

**Review by Bensk1 — APPROVED** [review]

**Pull request merged**